src/bloqade/task/bloqade.py

```diff
@@ -47,7 +83,22 @@ def run(
         nsteps: int = 2_147_483_647,
         interaction_picture: bool = False,
     ) -> "BloqadeTask":
-
+        """
```

Comment:
please also add one example to demonstrate how this function can be used. This is a user API.